taxi/views.py

```diff
@@ -34,6 +35,25 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+
+
+class ManufacturerUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Manufacturer
+    fields = "__all__"
+
+    def get_success_url(self, **kwargs) -> str:
+        return reverse_lazy("taxi:manufacturer-list")
```

Comment:
If your project predominantly uses simple success URL definitions that don't require custom logic, consider the inline comment approach for conciseness and clarity